### ISSUE TITLE
deeply nested conversations are unreadable (on mobile)

### ISSUE THREAD

**Issue by orthoxerox** [themotte/rDrama#191]
![Screenshot_20220817-232409](https://user-images.githubusercontent.com/8640149/185236207-a976e613-7c69-470d-8c46-8c3774cbcde2.png)

https://www.dev.themotte.org/post/98/look-at-me-i-am-the

Comments should have minimum width and force horizontal scrolling if they are too far to the right.

**Comment by orthoxerox** [comment]
They are barely readable on desktop, where horizontal scrolling is forced, but it's forced way too late.

**Comment by zorbathut** [comment]
This is an actual problem, yeah; I think Reddit's solution is to just open up a page on that comment instead of trying to expand the tree forever, and we should probably just adopt that solution.

**Comment by orthoxerox** [comment]
Hasn't this been fixed already? I see a "More comments" hyperlink after 9 comments now.

**Comment by zorbathut** [comment]
Nobody has fixed it. I'm curious what the different pathway is - maybe you're looking at it on a different page?

**Comment by orthoxerox** [comment]
No, same page, same browser, same device. The "more comments" link tries to open them inline on the desktop, but on mobile it takes me to a new page now.

**Comment by DinoInNameOnly** [comment]
This is not currently an issue as originally stated. I tested it and it shows "more comments" after 9 comments. One thing I noticed is that if I am the one who wrote all the comments, it doesn't make the "more comments" link until I refresh. Perhaps that is what you did to originally make this happen @orthoxerox ?

![306082283_416179640656278_7391918818139446447_n](https://user-images.githubusercontent.com/112996131/190580987-437d6b37-75ee-4af0-81ea-b6e51bd4f9ba.jpg)
